WTWSMS/common/nicknames/00_nicknames.txt

```diff
@@ -1647,20 +1660,27 @@ nick_the_quarreller = {
 nick_the_devil = {
 	allow = {
 		ruled_years = 10
-		trait = wroth
+		OR = {
+			trait = wroth
+			trait = cannibal_trait
+			has_character_modifier = cannibal
+		}
```

Comment:
That is why it is in the OR brackets. IF someone is a cannibal, they CAN be referred to as "the Devil". It does not mean they will, and it does not detract from the other qualifiers.